clients/ui/frontend/src/shared/hooks/useQueryParamNamespaces.ts

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { NamespaceSelectorContext } from '~/shared/context/NamespaceSelectorContext';
 import { isStandalone } from '~/shared/utilities/const';
-import { getNamespaceQueryParam } from '~/shared/api/apiUtils';
 import { useDeepCompareMemoize } from '~/shared/utilities/useDeepCompareMemoize';
 
 const useQueryParamNamespaces = (): Record<string, unknown> => {
   const { preferredNamespace: namespaceSelector } = React.useContext(NamespaceSelectorContext);
-  const namespace = isStandalone() ? namespaceSelector?.name : getNamespaceQueryParam();
+  // TODO: Readd GetNamespaceQueryParam once it's working
+  const namespace = isStandalone() ? namespaceSelector?.name : 'kubeflow';
```

Comment:
@lucferbux I made this change because `getNamespaceQueryParam` was not working in Kubeflow. When it gets the `window` the search url is not the same as what the dashboard has. We need to figure out a way to get the current namespace of the user.